titan/output.py

```diff
@@ -274,47 +274,22 @@ def print_components(
     """
     f = open(os.path.join(outdir, f"{run_id}_componentReport_ALL.txt"), "a")
 
-    race_count: Dict[str, int] = {}
-    race_list = []
-    header = ""
-    for race in races:
-        race_list.append(race)
-        header += "\t" + race
-
     # if this is a new file, write the header info
     if f.tell() == 0:
         f.write(
-            "run_id\trunseed\tpopseed\tt\tcompID\ttotalN\tNhiv\tNprep\tNtrtHIV"
-            "\tComp_Status\tPCA\tOral\tInjectable\tAware\tnidu\tcentrality\tdensity"
-            "\tEffectiveSize" + header + "\tdeg_cent\n"
+            "run_id\trunseed\tpopseed\tt\tcomponent"
+            "\tComp_Status\tPCA\tcentrality\tdensity"
+            "\tEffectiveSize\tdeg_cent\n"
         )
 
-    comp_id = 0
-    for comp in components:
-        for race in races:
-            race_count[race] = 0
-        assert comp.number_of_nodes() >= 0
-        tot_agents = (
-            nhiv
-        ) = ntrthiv = nprep = injectable_prep = oral = aware = pca = nidu = 0
+    for (id, comp) in enumerate(components):
+        pca = 0
+
         component_status = "control"
         trt_comp = trt_agent = False
 
         for agent in comp.nodes():
-            tot_agents += 1
-            race_count[agent.race] += 1
-            if agent.hiv.active:
-                nhiv += 1
-                if agent.random_trial.treated:
-                    ntrthiv += 1
-
-            if agent.prep.active:
-                nprep += 1
-                if agent.prep.type == "Inj":
-                    injectable_prep += 1
-                elif agent.prep.type == "Oral":
-                    oral += 1
-
+            # TO_REVIEW are these deivable from basic? add something to random trial's stats, maybe?
```

Comment:
Any follow-up to this thought?

Reply:
this is resolved in `moar performance`, we can wait to merge that in then do all at once? or I can also fix it here?